corehq/__init__.py

```diff
@@ -44,3 +44,7 @@ def _get_current_app():
 assert hasattr(_state.current_app, "_Proxy__local")
 object.__setattr__(_state.current_app, "_Proxy__local", _get_current_app)
 _state.get_current_app = _get_current_app
+
+if os.environ.get('DOCS_BUILD'):
+    import django
+    django.setup()
```

Comment:
What do you think of adding a comment above these lines to explain why this was done (similar to what's in the PR description)?

Reply:
Yeah it can't hurt. I initially assumed git history would be good enough to inspect this PR, but this warrants a comment. See d7e6743513ddf7d97b9f1bb3eadbf5fd67bcda33